src/features/auth/login-form/LoginForm.tsx

```diff
@@ -2,38 +2,105 @@ import TextField from "@mui/material/TextField";
 import Button from "@mui/material/Button";
 import { FunctionComponent } from "react";
 import useLoginFormState from "./use-login-form-state";
+import { Typography } from "@mui/material";
+import Check from "../Check/CheckKeepLogged.tsx";
+import Box from "@mui/material/Box";
+import useTheme from "@mui/material/styles/useTheme";
+import useMediaQuery from '@mui/material/useMediaQuery';
+import Link from '@mui/material/Link';
+import ucabLogo from "../images/ucabLogo.webp";
+import "./LoginForm.css";
+import { TextFieldTypography, 
+  UcabLogoStyles, 
+  ImageBoxStyles,
+  FormBottomStyles,
+  FormBottonStyles1,
+  FormBottomTypographyStyles,
+  ForgotPaswordStyles,
+  UnderFormStyles,
+  TextFieldStyles, 
+  ButtonStyles
+
+} from "./styles/LoginFormStyles.tsx";
 
 const LoginForm: FunctionComponent<Props> = ({ disabled, onSubmit }) => {
   const { email, password, onChangeEmail, onChangePassword } =
     useLoginFormState();
+    const theme = useTheme();
+    const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+    const FormBoxStyles = {
+        width: isSmallScreen ? '85%' : '350px',
+        height: '480px',
+        margin: 'auto', 
+        padding: isSmallScreen ? '18px' : '40px',
+        backgroundColor:'white',
+        display: 'flex',
+        flexDirection:'column',
+        alignItemns:'center',
+        justifyContent:'center',
+        borderRadius:'8px',
+        marginBottom:'16px',
+    };
 
   return (
     <>
-      <TextField
-        value={email}
-        label="Correo electronico"
-        type="email"
-        onChange={onChangeEmail}
-        disabled={disabled}
-      />
-      <TextField
-        value={password}
-        label="Contraseña"
-        type="password"
-        onChange={onChangePassword}
-        disabled={disabled}
-      />
-      <Button
-        variant="contained"
-        color="primary"
-        type="submit"
-        disabled={disabled}
-        onClick={() => {
-          onSubmit(email, password);
-        }}
-      >
-        Iniciar sesion
-      </Button>
+
+      <Box sx={FormBoxStyles}>
+        <Box sx={ImageBoxStyles}>
+          <img src={ucabLogo} alt="UCAB Logo" style={UcabLogoStyles} />
+        </Box>
+        <Box>
+          <Typography sx={TextFieldTypography}>Correo electronico</Typography>
+          <TextField
+            value={email}
+            label=""
+            type="email"
+            onChange={onChangeEmail}
+            disabled={disabled}
+            sx={TextFieldStyles}
+            InputProps={{ sx: { borderRadius: '11px' } }}
+          />
+          <Typography sx={TextFieldTypography}>Contraseña</Typography>
+          <TextField
+            value={password}
+            label=""
+            type="password"
+            onChange={onChangePassword}
+            disabled={disabled}
+            sx={{ ...TextFieldStyles, marginBottom: '26px' }}
+            InputProps={{ sx: { borderRadius: '11px' } }}
```

Comment:
Hay que homogenizar ese LOGIN_BORDER_RADIUS, como minimo deberia ser una constante.
```suggestion
            InputProps={{ sx: { borderRadius: LOGIN_BORDER_RADIUS } }}
          />
          <Typography sx={TextFieldTypography}>Contraseña</Typography>
          <TextField
            value={password}
            label=""
            type="password"
            onChange={onChangePassword}
            disabled={disabled}
            sx={{ ...TextFieldStyles, marginBottom: '26px' }}
            InputProps={{ sx: { borderRadius: LOGIN_BORDER_RADIUS } }}
```
Pero la forma correcta y una de las razones por las que estamos usando los estilos de MaterialUI es por su funcion de Templates que nos facilitarian la vida, entonces por que no homogenizar ese borderRadius alli?
```suggestion
          />
          <Typography sx={TextFieldTypography}>Contraseña</Typography>
          <TextField
            value={password}
            label=""
            type="password"
            onChange={onChangePassword}
            disabled={disabled}
            sx={{ ...TextFieldStyles, marginBottom: '26px' }}
```